content/en/real_user_monitoring/guide/retention_filter_best_practices.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Retention Filter Best Practices
+description: Learn the best practices for sequencing your retention filters to store the RUM data you need.
+private: true
+further_reading:
+  - link: '/real_user_monitoring/rum_without_limits/retention_filters'
+    tag: Documentation
+    text: Retention Filters
+  - link: '/real_user_monitoring/rum_without_limits/'
+    tag: Documentation
+    text: RUM without Limits
+  - link: '/real_user_monitoring/rum_without_limits/metrics'
+    tag: Documentation
+    text: Analyze Performance with Metrics
+---
+
+## Overview
+
+RUM without Limits allows you to capture all session data while only retaining the sessions that are valuable to your organization. This tool enhances your data management by separating session data ingestion from indexing.
+
+## Key features
+
+- **Dynamic retention filters**: Adjust what data to keep directly in the user interface
```

Comment:
```suggestion
- **Dynamic retention filters**: Adjust what data to keep directly in the Datadog user interface
```

Reply:
You might say "Adjust what data to keep without changing any code," since "keep directly in" made me expect a data storage location to come next in the sentence. But it was only a mild garden path, I understand why you would prefer to keep it as-is.

Reply:
@jhgilbert I think this is a fair point. I ended up changing it to "**Dynamic retention filters**: Adjust what data to keep without changing any code." Thank you for your feedback!